### PR TITLE
ECO-2477-[deploy error fixes]

### PR DESCRIPTION
**Pull request by Contentstack-AnkitaD**
Dynamic Import Removed

### CODE REVIEW

**Comment by abhishek305** [comment]
Add ticket id in title @Contentstack-AnkitaD

File: components/devtools.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { useState, useEffect } from 'react';
 import dynamic from 'next/dynamic';
 import Tooltip from './tool-tip';
```
Comment (Amitkanswal) [inline components/devtools.tsx:4]: pls run prettify here

**Review by Amitkanswal — CHANGES_REQUESTED** [review]

File: components/devtools.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React, { useState, useEffect } from 'react';
...
 
-const DynamicJsonViewer = dynamic(() => import('@textea/json-viewer').then((module) => ({ default: module.JsonViewer })), { ssr: false });
+import {JsonViewer} from "@textea/json-viewer"
```
Comment (abhishek305) [inline components/devtools.tsx:5]: semicolon missing

**Review by abhishek305 — CHANGES_REQUESTED** [review]
make sure double quotes are used instead single

**Review by abhishek305 — APPROVED** [review]

**Pull request merged**